src/babel.js

```diff
@@ -140,13 +140,21 @@ module.exports = () => {
 
   let proceed = false
 
+  let dataFromComment = {}
+
   return ({
     pre(state) {
       componentsList = new Set()
       pathsList = new Set()
       hashmap = {}
       /** don't apply reselector for files that were already proceed */
       proceed = (state.ast.comments || []).some(x => x.value.includes('__reselector__start__::'))
+
+      if (proceed) {
+        dataFromComment = JSON.parse(
```

Comment:
I think you can try to use this helper:

https://github.com/lttb/reselector/blob/29e4c7c509b45fe9eb4d124b67de7757fbf4c168/src/utils.js#L139